mini-e2e.groovy

```diff
@@ -18,6 +19,21 @@ node('cico-workspace') {
 			changelog: false
 	}
 
+	stage('skip ci/skip/e2e label') {
+		if (params.ghprbPullId == null) {
+			skip_e2e = 1
+			return
+		}
+		skip_e2e = sh(
+			script: "./scripts/get_github_labels.py --id=${ghprbPullId} --has-label=ci/skip/e2e",
```

Comment:
wouldn't it be easy to add more labels if we define the label as a variable?

Reply:
Maybe, but the groovy scripts would need adaption for that anyway. There is only one label at the moment, so no need to make it more flexible.